src/optimize/base_optimizer.js

```diff
@@ -365,12 +366,13 @@ export default class BaseOptimizer {
             'NODE_ENV': '"production"'
           }
         }),
-        new webpack.optimize.UglifyJsPlugin({
-          compress: {
-            warnings: false
-          },
+        new UglifyJsPlugin({
```

Comment:
I don't get it, If we don't care about mangling and compression, why minify in the first place? Removing this plugin would reduce the build time even further. 

All it does right now is white space removal, which can be accomplish by babel itself and its already in the webpack config.

Reply:
@vvigneshshanmugam thanks for the feedback. You should check the code on last master as we have described why we still need it: https://github.com/elastic/kibana/blob/master/src/optimize/base_optimizer.js#L396

Reply:
Thanks for the info, I didn't check the recent changes. Makes sense now 😃